error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -80,10 +82,23 @@ static final class Joining {
   }
 
   /** Prefer {@link Stream#empty()} over less clear alternatives. */
+  // XXX: We can additionally introduce a rule that maps `OptionalInt.empty().stream()` to
+  // `Intstream.empty()`, and likewise for `OptionalLong` and `OptionalDouble`, but those
+  // expressions are highly unlikely to be seen in the wild.
```

Comment:
... and in combination with the bug checker suggested in `IsEmpty`, these rules would be obsolete. So let's not do this.